index.js

```diff
@@ -0,0 +1,52 @@
+function getRandomIntInclusive(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+// console.log(getRandomIntInclusive(1,6));
+
+const dice = {
+	value : 1,
+	roll: function(){
+		this.value = getRandomIntInclusive(1,6);
+		return this.value;
```

Comment:
Well done, it is considered good practice to return a value even from object methods which mutate the object.